src/smartthings/smartapps/smartthings/amazon-echo-home-skill.src/amazon-echo-home-skill.groovy

```diff
@@ -190,7 +198,30 @@ def deviceAuthorization() {
 			section("Select Devices") {
 				input "switches", "capability.switch", multiple: true, required: false, title: "Selected Switches"
 				input "thermostats", "capability.thermostat", title: "Selected Thermostats", multiple: true, required: false
-				input "routinesEnabled", "bool", title: "Routines", options: ["All routines"], description: "Select routines", required: false
+				input "routinesEnabled", "bool", title: "Routines", options: ["All routines"], description: "", required: false, submitOnChange:true
+			}
+		}
+		if (isBlanketAuthorized() || areRoutinesEnabled()) {
+				def routines = location.helloHome?.getPhrases()
```

Comment:
Looks like just additional tab here and in a couple other places in this chunk - lines:

- 205
- 213
- 215
- 221 through 223